source/usage-examples/updateMany.txt

```diff
@@ -0,0 +1,26 @@
+=========================
+Update Multiple Documents
+=========================
+
+.. default-domain:: mongodb
+
+You can update multiple documents using the ``updateMany()`` method. 
+``updateMany()`` accepts a filter object, and updates the documents
+that matches the filter using a provided update document. The update
+document requires an :manual:`Update Operator
+</reference/operator/update>` to modify a field in a document.
+
+Create an `Object
+<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object>`_
+to specify additional options. Set the ``upsert`` option to ``true`` to create a new
+document if no documents match the filter.
```

Comment:
I think you should add a sentence here explaining how upsert will create a new document with any literal values provided in your filter, THEN apply your update changes to that new document. It's unintuitive behavior so i think it's valuable to point it out.

Reply:
Fixed - removed all instances of upsert